### PR TITLE
fix MSVC and MinGW compiler warnings

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**

### CODE REVIEW

File: libmscore/spanner.cpp [hunk 1]
```diff
@@ -605,7 +605,7 @@ void Spanner::computeEndElement()
                         Fraction tick = (l->ticks().ticks() == Lyrics::TEMP_MELISMA_TICKS) ? l->tick() : l->endTick();
                         Segment* s = score()->tick2segment(tick, true, SegmentType::ChordRest);
                         if (!s) {
-                              qDebug("%s no end segment for tick %d", name(), tick);
+                              qDebug("%s no end segment for tick %d/%d", name(), tick.numerator(), tick.denominator());
```
Comment (Jojo-Schmitz) [inline libmscore/spanner.cpp]: or use `qDebug("%s no end segment for tick %d", name(), tick.ticks();` instead?

Comment (MarcSabatella) [reply libmscore/spanner.cpp]: I actually changed this to tick.ticks() in another or yesterday, so if you do the same, that would lessen the chance of a merge conflict I assume.

Comment (Jojo-Schmitz) [reply libmscore/spanner.cpp]: done, seen it being used elsewhere

Comment (Jojo-Schmitz) [reply libmscore/spanner.cpp]: Done the same way in #4730

**Pull request merged**